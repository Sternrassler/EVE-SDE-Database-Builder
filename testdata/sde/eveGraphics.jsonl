{"graphicID":20,"graphicFile":"res:/UI/Texture/Icons/1_64_1.png","description":"Generic icon 1","obsolete":0,"graphicName":"icon01_01","graphicType":"Icon"}
{"graphicID":21,"graphicFile":"res:/UI/Texture/Icons/1_64_2.png","description":"Generic icon 2","obsolete":0,"graphicName":"icon01_02","graphicType":"Icon"}
{"graphicID":22,"graphicFile":"res:/UI/Texture/Icons/1_64_3.png","description":"Generic icon 3","obsolete":0,"graphicName":"icon01_03","graphicType":"Icon"}
