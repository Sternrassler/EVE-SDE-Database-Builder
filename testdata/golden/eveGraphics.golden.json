[
  {
    "graphicID": 20,
    "graphicFile": "res:/UI/Texture/Icons/1_64_1.png",
    "description": "Generic icon 1"
  },
  {
    "graphicID": 21,
    "graphicFile": "res:/UI/Texture/Icons/1_64_2.png",
    "description": "Generic icon 2"
  },
  {
    "graphicID": 22,
    "graphicFile": "res:/UI/Texture/Icons/1_64_3.png",
    "description": "Generic icon 3"
  }
]